client_test/virtualfund_benchmark_test.go

```diff
@@ -0,0 +1,93 @@
+package client_test
+
+import (
+	"math/big"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/statechannels/go-nitro/client"
+	"github.com/statechannels/go-nitro/client/engine/chainservice"
+	"github.com/statechannels/go-nitro/client/engine/messageservice"
+	"github.com/statechannels/go-nitro/protocols/virtualfund"
+	"github.com/statechannels/go-nitro/types"
+)
+
+// TestBenchmark sets up three clients, then runs a virtual funding benchmark, printing the duration
+// to the screen.
+func TestBenchmark(t *testing.T) {
+
+	logFile := "virtualfund_benchmark_test.log"
+	truncateLog(logFile)
+
+	chain := chainservice.NewMockChain()
+	broker := messageservice.NewBroker()
+
+	clientAlice := setupClient(aliceKey, chain, broker, logFile)
+	clientBob := setupClient(bobKey, chain, broker, logFile)
+	clientIrene := setupClient(ireneKey, chain, broker, logFile)
+
+	directlyFundALedgerChannel(t, clientAlice, clientIrene)
+	directlyFundALedgerChannel(t, clientIrene, clientBob)
+
+	done := make(chan bool)
```

Comment:
Since we're not actually using the boolean value we could just use an empty struct channel.
```
		done := make(chan struct{})
```
It doesn't really matter here but it's more [performant](https://dave.cheney.net/2014/03/25/the-empty-struct) and removes ambiguity around true/false values on the channel.

Reply:
Good to know!

Reply:
https://github.com/statechannels/go-nitro/pull/354/commits/9cb6ca7416caa30a0734680e4acd7fc905c85095